po/de.po

```diff
@@ -16,32 +16,31 @@ msgstr ""
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=n != 1;\n"
-"X-Generator: Poedit 3.4.2\n"
+"X-Generator: Poedit 3.5\n"
 
 #.
 #: src/widgets/sidebar.vala:32
 msgid "Only display the runners that are currently installed on your system"
 msgstr ""
+"Nur Ausführungswerkzeuge anzeigen die aktuell auf dem System installiert sind"
 
 #.
 #: src/widgets/sidebar.vala:35
-#, fuzzy
 msgid "Only Installed"
 msgstr "Nur installierte"
 
 #.
 #: src/widgets/info-box.vala:25
 msgid "_About ProtonPlus"
-msgstr ""
+msgstr "_Über ProtonPlus"
 
 #: src/widgets/info-box.vala:181 src/widgets/info-box.vala:184
 msgid "Load more"
 msgstr "Mehr laden"
 
 #: src/widgets/info-box.vala:196 src/widgets/info-box.vala:332
-#, fuzzy
 msgid "There was an error while fetching data from the GitHub API."
-msgstr "Beim Abrufen von Daten von der GitHub-API ist ein Fehler aufgetreten"
+msgstr "Beim Abrufen von Daten von der GitHub-API ist ein Fehler aufgetreten."
```

Comment:
Below this line is a line that might not be correct (I can't click the specific change):

```
msgid "Cancel the installation"
msgstr "Breche Installation ab"
```

This may not be the best translation? It's the tooltip when you hover the "X" button to cancel a pending download.

Isn't it more correct to say `Abbrechen der Installation` or `Installation abbrechen`?

The current translation feels like you used it like a sentence fragment like `Cancel installation of X` but it's not used that way in the GUI so that would be incorrect. :)

Reply:
done